scripts/install-windows-prereqs.ps1

```diff
@@ -12,8 +12,8 @@ Param(
     # We skip the hash check for the vs_buildtools.exe file because it is regularly updated without a change to the URL, unfortunately.
     [string]$VSBuildToolsURL = 'https://aka.ms/vs/15/release/vs_buildtools.exe',
     [string]$VSBuildToolsHash = '',
-    [string]$Clang7URL = 'http://releases.llvm.org/7.0.1/LLVM-7.0.1-win64.exe',
-    [string]$Clang7Hash = '672E4C420D6543A8A9F8EC5F1E5F283D88AC2155EF4C57232A399160A02BFF57',
+    [string]$Clang8URL = 'https://releases.llvm.org/8.0.0/LLVM-8.0.0-win64.exe',
```

Comment:
There's a 8.0.1, which is the latest 8.x at the moment it seems, should we use that?